syntax = "proto3";

message Request{
  string RFW_ID =1;
  string  benchmark_type = 2;
  string work_metric =3;
  int32  batch_unit =4;
  int32 batch_ID= 5;
  int32 batch_size =6;
}

message Data_Response{
  string RFW_ID = 1;
  int32 last_batch_ID = 2;
  repeated batches batch_data =3;
}

message batches {
  repeated float sample_data = 1;
}